docs/UserGuide.md

```diff
@@ -111,8 +111,83 @@ You should notice the GUI of the application pop up.
 3. For advanced users, view all feature details [here]().
 
 --------------------------------------------------------------------------------------------------------------------
+# Introducing LinkMeIn: A Quick Tour #
```

Comment:
Great description! But upon second look, we might want to keep it short as well to prevent it from being wordy. 

2nd para and 3rd para might consider doing bullet points?